### DISCUSSION TITLE
基于有赞微信小程序改造的可以同时在Taro3-React和React中使用的组件库

### DISCUSSION THREAD

**Discussion by hisanshao** [youzan/vant#9792] [General]
1: 我们认为它可以为vant锦上添花[文档查看](https://antmjs.github.io/vantui/#/home)
2: 我们会持续同步有赞小程序的最新版，目前同步至最新的1.9.2
3: 希望可以在Community Ecosystem中加入我们的仓库

**Comment by chenjiahan** [comment]
很棒~ 我加一下

**Reply by hisanshao** [reply]
感谢感谢，为大家做贡献

**Reply by chenjiahan** [reply]
已添加~ https://github.com/youzan/vant/pull/9798
